components/docs-chef-io/content/automate/infra_server.md

```diff
@@ -126,6 +126,27 @@ Installations require elevated privileges, so run the commands as the superuser
 
 1. [Set up `knife`]({{< relref "infra_server.md#use-knife-with-chef-infra-server" >}}) for use with Chef Infra Server.
 
+1. To send data from the Chef Infra Server to an external Chef Automate installation,
+   first create a `patch.toml` file that contains the configuration stanza:
+
+    ```toml
+    [global.v1.external.automate]
+    enable = true
+    node = "https://<automate server url>"
+    [global.v1.external.automate.auth]
+    token = "<data-collector token>"
+    [global.v1.external.automate.ssl]
+    server_name = "<server name from the automate server ssl cert>"
+    root_cert = """<pem format root CA cert>
+    """
+    [auth_n.v1.sys.service]
+    a1_data_collector_token = "<data-collector token>"
```

Comment:
The variable name is misleading; it's fine to use a data collector token from A2 here. Not sure if calling that out would be more confusing than not, though.

Reply:
You're right, it is confusing. Is renaming the variable possible? Outside of docs files, I see it called 10 times.